### ISSUE TITLE
Timeouts in MessageRejectReportComputedTask

### ISSUE THREAD

**Issue by lukasz-glen** [golemfactory/golem-messages#13]
Make sure that the message MessageRejectReportComputedTask supports setting timeouts as a reason of rejection (there is appriopriate logic). 
Investigate if timeouts for subtask and task are distinguished in existing code. If it is not documentation should be corrected (LG will do).
https://docs.google.com/document/d/1QMnamlNnKxichfPZvBDIcFm1q0uJHMHJPkCt24KElxc/edit?pli=1#heading=h.dhlbf3h55z2u

**Comment by jiivan** [comment]
Already done by @etam ?

**Comment by jiivan** [comment]
Investigate `ComputeTaskDef`/`MessageTaskToCompute` for task/subtask timeouts. What exactly is `CTD` deadline?

**Comment by jiivan** [comment]
Both timeouts (from subtask and task) are unified in https://github.com/golemfactory/golem/blob/237b3ea76600c62c5d58cc7f5edb8389235d326b/golem/task/taskcomputer.py#L341 and further processed in https://github.com/golemfactory/golem/blob/237b3ea76600c62c5d58cc7f5edb8389235d326b/golem/task/taskcomputer.py#L179

I also added some clarifying comments in connected PR

tl;dr; timeouts are *not* distinguished
